docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
+| application-a   | production     | 12c      | db-connection       | connectionstring-a-prod           |
+| application-a   | development    | 23cbeta  | db-dev-name         | databasename-a-dev                |
+| application-a   | development    | 23cbeta  | db-dev-connection   | connectionstring-a-dev            |
+| application-b   | production     | 19c      | db-name             | databasename-b-prod               |
+| application-b   | production     | 19c      | db-connection       | connectionstring-b-prod           |
+| application-b   | development    | 23cbeta  | db-dev-name         | databasename-b-dev                |
+| application-b   | development    | 23cbeta  | db-dev-connection   | connectionstring-b-dev            |
+| application-c   | secret         | 23.4     | json-db             | 23c-json-db                       |
+| application-c   | secret         | 23.4     | json-sdb-conn       | 23c-mongo-conn                    |
+| application-c   | secret         | 23.4     | txenventq           | 23c-conn-string                   |
+| application-c   | secret         | 23.4     | txeventq            | 23c-kafka-name                    |
+```
+
+## Config Server REST endpoints overview
+
+The following REST Endpoints are available to Config Server entries. The table lists which minimum required role that is needed to perform the operation. `N/A` in the table below means that endpoint doesn't require authentication to be accessed.
+
+| End point                     | Method | Description                                             | Minimum Required Role |
+|-------------------------------|--------|---------------------------------------------------------|-----------------------|
+| /srv/config/all               | GET    | Get all distinct properties for a service (application) | N/A                   |
+| /srv/config/properties        | GET    | Get all distinct properties with filters (see examples) | N/A                   |
+| /srv/config/properties/add    | POST   | Create properties from a file                           | ROLE_USER             |
+| /srv/config/property/add      | POST   | Create a property                                       | ROLE_USER             |
+| /srv/config/property/update   | PUT    | Update a property                                       | ROLE_USER             |
+| /srv/config/properties/delete | DELETE | Delete properties with filters (see examples)           | ROLE_ADMIN            |
+
+### Config Server REST endpoints examples
+
+In all examples below you need to replace `<username>:<password>` with your username and password when necessary. ([Getting User information](../../security/azn-server/)). The examples are using `curl` to interact with the REST endpoints. They also requires that you have opened a tunnel on port 8080 to either the `config-server` or `obaas-admin` service. For example:
```

Comment:
Change to:

In all of the following examples, replace ... necessary. ... The examples ... the REST API endpoints. This also requires an open tunnel on ...